index.html

```diff
@@ -119,6 +120,31 @@ <h3>
           whether to show UI that triggers its use.
         </div>
         <section>
+          <h4>
+            <dfn>canShare()</dfn> method
+          </h4>
+          <p>
+            When the <a>canShare()</a> method is called with argument
+            <var>data</var>, run the following steps:
+          </p>
+          <ol data-link-for="ShareData">
+            <li>If none of <var>data</var>'s members <a>title</a>, <a>text</a>,
+            <a>url</a> or <a>files</a> are present, return <code>false</code>.
+            </li>
+            <li>return <code>true</code>.
+            </li>
+          </ol>
+          <div class="note" data-link-for="ShareData">
+            <p>
+              <code>canShare()</code> MUST return <code>false</code> whenever
```

Comment:
This is normative, and therefore shouldn't be in a note. (I must've missed this text last time I looked.)

This is actually not true, as stated. If `url` is invalid, `share` should reject with `TypeError`, but `canShare` should return true because it doesn't check URL. I think `canShare` *should* return false if `url` is invalid, and thus this statement here better matches what I want than the definition above.

I think I'd be happier with this being the *definition* of `canShare`, rather than a corollary (so we don't have to duplicate or abstract out the checking logic). Do you think we can get away with simply deleting the steps, and writing "`canShare()` MUST return false whenever ..... It MUST return true otherwise." ?